cg/meta/orders/ticket_handler.py

```diff
@@ -23,12 +23,12 @@ def __init__(self, osticket_api: OsTicket, status_db: Store):
         self.status_db: Store = status_db
 
     @staticmethod
-    def parse_ticket_number(name: str) -> Optional[int]:
+    def parse_ticket_number(name: str) -> Optional[str]:
         """Try to parse a ticket number from a string"""
         # detect manual ticket assignment
         ticket_match = re.fullmatch(r"#([0-9]{6})", name)
         if ticket_match:
-            ticket_number = int(ticket_match.group(1))
+            ticket_number = ticket_match.group(1)
```

Comment:
```suggestion
            ticket_id: Optional[str] = ticket_match.group(1)
```
Or? ticket_number feels confusing (one would assume int and not str)

Reply:
Good point